unstructured/utils.py

```diff
@@ -50,3 +51,26 @@ def dependency_exists(dependency):
     except ImportError:
         return False
     return True
+
+
+# Copied from unstructured/ingest/connector/biomed.py
+def validate_date_args(date):
+    date_formats = ["%Y-%m-%d", "%Y-%m-%dT%H:%M:%S", "%Y-%m-%dT%H:%M:%S%z"]
+
+    valid = False
+    if date:
+        for format in date_formats:
+            try:
+                datetime.strptime(date, format)
+                valid = True
+                break
+            except ValueError:
+                pass
+
+        if not valid:
```

Comment:
if we return per above comment, we don't need the `if not valid`, can just raise the `ValueError`

Reply:
(sorry for the nits on the copied function, probably better if functions in utils.py have a bit of a higher bar than one-off connector functions)

Reply:
@cragwolfe My thinking for putting this in utils is that we have a single set of input args --start-date --end-date, that this method would validate the args, and then each connector that takes a date range would then translate date time into the format the connector would use.